packages/client/hmi-client/src/components/workflow/ops/funman/tera-funman-node.vue

```diff
@@ -107,4 +127,20 @@ watch(
 	},
 	{ immediate: true }
 );
+
+watch(
+	() => [inProgressId.value, currentProgress.value],
+	() => {
+		if (inProgressId.value) {
+			StartRequestTimer();
+		} else {
+			message.value = '';
+			const state = _.cloneDeep(props.node.state);
+			state.isRequestUnresponsive = false;
+			emit('update-state', state);
+			clearTimeout(timer.value);
+		}
+	},
+	{ immediate: true, deep: true }
+);
```

Comment:
This feels more complicated than it needs to be. I think we should get by with just the poller since it pretty much provide the same information as the additional watcher that is being introduced here.

Also if you are using a setTimeout you should dismiss it in beforeUnmount or onUnmount hook.